common/djangoapps/student/views.py

```diff
@@ -1804,6 +1805,30 @@ def create_account_with_params(request, params):
     return new_user
 
 
+def _enroll_user_in_pending_courses(student):
+    """
+    Enroll student in any pending courses he/she may have.
+    """
+    if not student:
+        log.error(u"There is no student")
```

Comment:
Error message should be more descriptive. If you saw this error in the logs with minimal context, what information would you need to understand what actually went wrong?

I'm still not convinced that this line would ever actually be executed. Part of the point of logging this is to prove this belief so that this can eventually be removed